packages/jsActions/nanoflow-actions-native/CHANGELOG.md

```diff
@@ -4,3 +4,8 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.1.0] Nanoflow Commons - 2021-9-28
```

Comment:
Similar question as https://github.com/mendix/widgets-resources/pull/953#discussion_r718267691

Reply:
I gave an explanation in the mentioned PR. For native it makes sense to keep the module name in the module changelogs, because we also include the widgets changelogs in the module changelogs. So it's easy to distinguish :)

Reply:
Okay!